api-server/src/homepage/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+</head>
+
+<body>
+  <div class="overlord">
```

Comment:
Unused class?

Reply:
Yeah I think that was from some testing that I forgot about.